README.md

```diff
@@ -20,127 +19,304 @@ let thirdNumber = numbers[2]
- 
+
  if rainbowColors.count > 5 {
     print("There are more than 5 colors in a rainbow.")
  }
  //prints "There are more than 5 colors in a rainbow."
 ```
 
-* Referring to `isEmpty` - I like the idea of stepping them through the following two examples:
+- Referring to `isEmpty` - I like the idea of stepping them through the following two examples:
 
 ![deliLine](http://i.imgur.com/ivgaSon.png?1)
+
 ```swift
  var deliLine = [
     "Albert Einstein",
     "Isaac Newton",
     "Galileo Galilei",
     "Marie Curie"
  ]
- 
+
  if deliLine.isEmpty {
     print("The deli line is empty")
  } else {
     print("There are people waiting to eat!")
  }
- 
+
  // prints "There are people waiting to eat!"
 ```
+
 ![Moons](http://i.imgur.com/reNsnn6.png?1)
+
 ```swift
  let moonsOfJupiter = [
     "Europa",
     "Ganymedge",
     "Io",
     "Callisto"
 ]
- 
+
  if !moonsOfJupiter.isEmpty {
     for moon in moonsOfJupiter {
         print("\(moon) is a moon orbiting Jupiter.")
     }
  }
- 
+
  // Europa is a moon orbiting Jupiter.
  // Ganymede is a moon orbiting Jupiter.
  // Io is a moon orbiting Jupiter.
  // Callisto is a moon orbiting Jupiter.
 ```
 
+--------------------------------------------------------------------------------
+
+In Unit 1 you saw the following:
+
+```swift
+let greeting = "Hello"
+let loudGreeting = greeting.uppercaseString
+
+print(loudGreeting)
+// prints "HELLO"
+```
+
+This was an example of a 'method', which can be thought of as a pre-written function that helps you accomplish common tasks.
+
+You also saw `.append` in the last lesson, which is another method. Whilst there are some common methods, many are contextual and only work on particular types. For example, a method like `.append` that works on an array will not work for a String, and `.uppercaseString` will not work on an array.
+
+The `.append()` method adds a value to the end of an array. You could achieve this in several ways which would involve far more code and calculation, I think you'll agree that `.append()` is far more convenient. There are many other methods for arrays, and in this lesson you'll look at some others.
+
+## Enumeration
+
+Before going any further, there's one other array concept to introduce.
+
+Say you have an array of string literals which represent a list of instructions to create great tea. If you want to print these instructions, you would need to do the following:
+
+```swift
+var toMakeTea = [
+    "Boil Water",
+    "Add tea bag to cup",
+    "Wait ten minutes",
+    "Add Milk",
+    "Drink"
+]
+
+print("1\. \(toMakeTea[0])")
+print("2\. \(toMakeTea[1])")
+// and so on
+
+// prints "1\. Boil Water"
+// prints "2\. Add tea bag to cup"
+// etc
+```
+
+This way of accomplishing the task is incredibly tedious. Knowing how a `for-in` loop works, you can iterate over each value in the array of strings. You also the index of the value in the array because you want to display numbered instructions.
+
```

Comment:
@JimCampagno can we actually show this using a `for-in` loop? Or what the code for the above phrase would be?